src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/README.md

```diff
@@ -0,0 +1,79 @@
+# `Raw Atlas Slicing`
+
+## Overview
+
+Raw Atlas Slicing is the process of assigning country codes to all Atlas features given an intermediary raw Atlas file. As input, the process takes in a raw Atlas and produces a sliced raw Atlas. It's important to note that both the input and output Atlas files will contain only Points, Lines and Relations. One of the main advantages of the country slicing process is that it will leverage the `DynamicAtlas` and attempt to build Relations that span multiple shards in their entirety. Once the Relation is built, it can be properly sliced along the given country boundaries and as a result, each country can contain valid, country-specific portions of multi-polygons that span one or more countries. In accomplishing this, we can achieve full OSM parity and be able to ingest OSM administrative boundaries and coast lines. Both of these features are currently left out of the Atlas.
+
+## Terminology
+
+This section calls out the main terms and concepts that are key to the slicing code.
+
+* We rely on the JTS (Java Topology Suite) library for all geometric operations (spatial queries, intersection and cutting requests) and some key data structures (R-tree, JTS geometry representations).
+* The `CountryBoundaryMap` class is the main driver behind all slicing operations. This class contains the raw country boundary `MultiPolygon` representations as well as an underlying optimization (the grid index) that allows for quick lookup to see whether a specific feature is contained in a two dimensional rectangle.   
```

Comment:
to see whether a specific feature is contained in a two dimensional rectangle. -> in a boundary